using-timescaledb/data-retention.md

```diff
@@ -147,72 +69,58 @@ SELECT remove_retention_policy('conditions');
 ```
 
 For more information, please see the [API documentation][add_retention_policy].
 
+### Data Retention with Continuous Aggregates [](retention-with-aggregates)
 
-### Using External Job Schedulers
-
-While the built-in scheduling framework automates data retention, the
-`drop_chunks` command can be combined with an external tool for job scheduling,
-like `crontab` or `systemd`, to schedule such commands.
-
-#### Using Crontab
-
-The following cron job will drop chunks every day at 3am:
-
-```bash
-0 3 * * * /usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');" >/dev/null 2>&1
-```
-
-The above cron job can easily be installed by running `crontab -e`.
-
-
-#### Using a Systemd Timer
-
-On a systemd-based OS (most modern Linux distributions), a systemd [unit][] with
-accompanying unit [timer][] can also be used to implement a
-retention policy.
+Extra care must be taken when using retention policies or `drop_chunks` calls on
+hypertables which have [continuous aggregates][continuous_aggregates] defined on
+them. Similar to a refresh of a materialized view, a refresh on a continuous aggregate
+will update the aggregate to reflect changes in the underlying source data. This means
+that any chunks that are dropped in the region still being refreshed by the
+continuous aggregate will cause the chunk data to disappear from the aggregate as
+well. If the intent is to keep the aggregate while dropping the underlying data,
+the interval being dropped should not overlap with the offsets for the continuous
+aggregate.
 
-First, create, e.g., `/etc/systemd/system/retention.service` unit file:
-
-```ini
-[Unit]
-Description=Drop chunks from the 'conditions' table
+As an example, let's add a continuous aggregate to our `conditions` hypertable:
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_daily (day, device, temp)
+WITH (timescaledb.continuous) AS
+SELECT time_bucket('1 day', time), device, AVG(temperature)
+FROM conditions
+GROUP BY (1, 2);
 
-[Service]
-Type=oneshot
-ExecStart=/usr/bin/psql -h localhost -p 5432 -U postgres -d postgres -c "SELECT drop_chunks('conditions', INTERVAL '24 hours');"
+SELECT add_continuous_aggregate_policy('conditions_summary_daily', '7 days', '1 day', '1 day');
 ```
 
-Then create a timer to run this unit, e.g., `/etc/systemd/system/retention.timer`:
-
-```ini
-[Unit]
-Description=Run data retention at 3am daily
-
-[Timer]
-OnCalendar=*-*-* 03:00:00
+This will create the `conditions_summary_daily` aggregate which will store the daily
+temperature per device from our `conditions` table. However, we have a problem here
+if we're using our 24 hour rentention policy from above, as our aggregate will capture
+changes to the data for up to seven days. As a result, we will update the aggregate
+when we drop the chunk from the table, and we'll ultimately end up with no data in our
+`conditions_summary_daily` table.
```

Comment:
Hmmm... usually it is not recommended to show incorrect examples since misunderstanding are quite common. It is usually more common to show a correct example and then digress into the problematic case: along the lines "Note that ... if we did it this way instead, it would not work since...". Then you keep the "faulty" code located to the discussion about it.